docs/AddingDoppl.md

```diff
@@ -0,0 +1,565 @@
+
+You should see a block of constants for some version numbers that are used
+for multiple artifacts:
+
+```groovy
+def supportVersion = "26.1.0"
+def archVer = "1.0.0-rc1"
+def retroVer = "2.3.0"
+```
+
+Add these two to that block:
+
+```groovy
+def dopplArchVer = "1.0.0.0-rc1"
+def dopplRetroVer = "2.3.0.7"
+```
+
+### Add Some doppl Statements
+
+Then, replace the existing `dependencies` closure with the following:
+
+```groovy
+dependencies {
+  implementation      "com.android.support.constraint:constraint-layout:1.0.2"
+  implementation      "com.android.support:recyclerview-v7:$supportVersion"
+  implementation("com.android.support:support-v4:$supportVersion") {
+    exclude group: 'com.android.support', module: 'support-media-compat'
+  } // for https://issuetracker.google.com/issues/64909326
+  implementation      "com.squareup.picasso:picasso:2.5.2"
+
+  implementation      "io.reactivex.rxjava2:rxjava:2.1.5"
+  doppl               "co.doppl.io.reactivex.rxjava2:rxjava:2.1.5.0"
+  implementation      "io.reactivex.rxjava2:rxandroid:2.0.1"
+  doppl               "co.doppl.io.reactivex.rxjava2:rxandroid:2.0.1.2"
+
+  implementation      "com.squareup.retrofit2:converter-gson:$retroVer"
+  implementation      "com.squareup.retrofit2:adapter-rxjava2:$retroVer"
+  doppl               "co.doppl.com.squareup.retrofit2.urlsession:retrofit:$dopplRetroVer"
+  doppl               "co.doppl.com.squareup.retrofit2.urlsession:converter-gson:$dopplRetroVer"
+  doppl               "co.doppl.com.squareup.retrofit2.urlsession:adapter-rxjava2:$dopplRetroVer"
+
+  implementation      "android.arch.lifecycle:runtime:1.0.3"
+  doppl               "co.doppl.android.arch.lifecycle:runtime:$dopplArchVer"
+  implementation      "android.arch.lifecycle:extensions:$archVer"
+  doppl               "co.doppl.android.arch.lifecycle:extensions:$dopplArchVer"
+  implementation      "android.arch.lifecycle:reactivestreams:$archVer"
+  annotationProcessor "android.arch.lifecycle:compiler:$archVer"
+
+  testImplementation  "junit:junit:4.12"
+  testImplementation  "co.doppl.lib:androidbasetest:0.8.5"
+  testDoppl           "co.doppl.lib:androidbasetest:0.8.5.0"
+}
+```
+
+The first few lines are for libraries tied to our UI, such as `recyclerview-v7`.
+Our code that is being converted to Objective-C does not refer to those libraries,
+so we do not need to make any changes there.
+
+For our RxJava, RxAndroid, Retrofit, and Architecture Components dependencies,
+we need to add `doppl` lines to tell Doppl what [pre-converted libraries](./Libraries)
+we should use. Technically, there is nothing magic about having the `doppl`
+statements alongside their `compile` equivalents, but keeping them together
+helps us ensure that we have added all the necessary `doppl` statements.
+In many cases, there is a 1:1 mapping between the `compile` statement
+and the corresponding `doppl` statement.
+
+The pair of `co.doppl.lib:androidbasetest` dependencies set up the framework
```

Comment:
Maybe link to https://github.com/doppllib/doppllib.github.io/blob/master/docs/Testing.md#tests-and-testdoppl if readers want to know more why you need both `testImplementation` and `testDoppl`. Otherwise, it's confusing that there's no `doppl` alternative for `junit` and, more so, that there's now a doppl dependency being pulled in with the non-doppl configuration `testImplementation`.

Reply:
I added the link &mdash; thanks!